lib/galaxy/webapps/galaxy/api/__init__.py

```diff
@@ -0,0 +1,84 @@
+"""
+This module *does not* contain API routes. It exclusively contains dependencies to be used in FastAPI routes
+"""
+from typing import (
+    Optional,
+)
+
+from fastapi import (
+    Cookie,
+    Depends,
+    Header,
+    HTTPException,
+    Query,
+)
+from sqlalchemy.orm import Session
+
+from galaxy import (
+    app as galaxy_app,
+    exceptions,
+    model,
+)
+from galaxy.app import UniverseApplication
+from galaxy.managers.session import GalaxySessionManager
+from galaxy.managers.users import UserManager
+from galaxy.model import User
+from galaxy.work.context import SessionRequestContext
+
+
+def get_app() -> UniverseApplication:
+    return galaxy_app.app
+
+
+def get_db(app: UniverseApplication = Depends(get_app)) -> Session:
+    # TODO: return sqlachemy 2.0 style session without autocommit and expire_on_commit!
+    return app.model.session
+
+
+def get_user_manager(app: UniverseApplication = Depends(get_app)):
+    return UserManager(app)
+
+
+def get_session_manager(app=Depends(get_app)) -> GalaxySessionManager:
```

Comment:
Nice to see some type annotations kicking in! :)
```suggestion
def get_session_manager(app: UniverseApplication = Depends(get_app)) -> GalaxySessionManager:
```